### PR TITLE
Rename cli errata tests as per issue #3083

### PR DESCRIPTION
**Pull request by sthirugn**

### CODE REVIEW

File: tests/foreman/cli/test_errata.py [hunk 1]
```diff
@@ -372,25 +374,7 @@ def test_errata_list_1(self):
         """
 
     @stubbed()
-    def test_errata_list_2(self):
```
Comment (oshtaier) [inline tests/foreman/cli/test_errata.py]: Specific reason for that?

**Comment by oshtaier** [comment]
ACK pending question

File: tests/foreman/cli/test_errata.py (hunk 1 above)
Comment (sthirugn) [inline tests/foreman/cli/test_errata.py]: Yes, this is a dupe and covered by another test `test_positive_list_filter_by_product_name`

**Comment by sthirugn** [comment]
Question answered and rebased.

**Comment by oshtaier** [comment]
thanks

**Pull request merged**